snapshot/Makefile

```diff
@@ -46,6 +50,15 @@ container: controller provisioner container-quick
 container-quick:
 	cp _output/bin/snapshot-controller deploy/docker/controller
 	cp _output/bin/snapshot-provisioner deploy/docker/provisioner
+	# Copy the root CA certificates -- cloudproviders need them
+	cp --parent -r /etc/pki/ca-trust deploy/docker/controller
```

Comment:
this does not always work especially if ca-certificates pkg is not locally installed

Reply:
this Dockerfile seems to work for me locally
```
FROM  busybox:glibc
CMD rpm -i https://www.rpmfind.net/linux/fedora/linux/development/rawhide/Everything/x86_64/os/Packages/c/ca-certificates-2017.2.20-6.fc28.noarch.rpm
CMD EXTRACT_UNSAFE_SYMLINKS=1 rpm -i ca-certificates-2017.2.20-6.fc28.noarch.rpm
```